.github/workflows/tanzu-tile.yml

```diff
@@ -0,0 +1,56 @@
+name: Tanzu Tile
+
+# The workflow triggered by any change in deployments/cloudfoundry/bosh/
+# or /deployments/cloudfoundry/tile
+
+on:
+  pull_request:
+    paths:
+      - 'deployments/cloudfoundry/bosh/**'
+      - 'deployments/cloudfoundry/tile/**'
+
+permissions:
+  contents: write
+
+defaults:
+  run:
+    working-directory: 'deployments/cloudfoundry/tile'
+
+jobs:
+
+  test:
+    name: Test Tanzu Tile creation
+    # Use 20.04.5 until https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/16450 is resolved
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v3
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+      - name: Install BOSH CLI
+        shell: bash
+        run: |
+          brew install cloudfoundry/tap/bosh-cli
+          bosh -v
+      - name: Install PCF CLI
+        shell: bash
+        run: |
+          pip install tile-generator
+      - name: Run release script
+        shell: bash
+        run: |
+          ./make-latest-tile
+          FILE=""
+          # The tile will be in the product directory, named
+          # splunk-otel-collector-VERSION.pivotal
+          for f in $(find product -maxdepth 1 -regex "product\/splunk-otel-collector\-.*pivotal")
```

Comment:
I think something like this could be more straightforward?
```
size="$(stat -c '%s' product/splunk-otel-collector-*.pivotal)"
if [[ $size -eq 0 ]]; then
  echo "File is empty!" >&2
  exit 1
fi
```

Reply:
Let me try that. The reason it's more complicated was so that it could handle a regex file name, but if stat works as well it's a lot simpler.

Reply:
Looks like it worked, thanks!